lms/templates/student_account/form_field.underscore

```diff
@@ -32,10 +32,11 @@
                     data-errormsg-<%- type %>="<%- msg %>"
             <%  });
             } %>
-            <% if ( required ) { %> aria-required="true" required<% } %>>
-        <% _.each(options, function(el) { %>
-            <option value="<%- el.value%>"<% if ( el.default ) { %> data-isdefault="true" selected<% } %>><%- el.name %></option>
-        <% }); %>
+            <% if ( required ) { %> aria-required="true" required<% } %>
+        >
+            <% _.each(options, function(el) { %>
+                <option value="<%- el.value%>"<% if ( el.default ) { %> data-isdefault="true" selected<% } %>><%- el.name %></option>
+            <% }); %>
```

Comment:
There's no actual changes here: formatting nits because this was getting incredibly hard to read when I had to read it.